Makefile

```diff
@@ -42,12 +47,21 @@ help:
 	@echo '  TIMEOUT           Maximum timeout to wait when bringing cluster up and down? [default=300]'
 	@echo '  VENV_NAME         Which Python virtual environment name to use? [default=reana]'
 	@echo '  DEMO              Which demo example to run? [e.g. reana-demo-helloworld; default=several runable examples]'
+	@echo '  CLUSTER_CONFIG    REANA Cluster environment mode.'
 	@echo
 	@echo 'Examples:'
 	@echo
 	@echo '  # how to set up personal development environment:'
 	@echo '  $$ GITHUB_USER=johndoe MINIKUBE_DRIVER=virtualbox make setup clone'
 	@echo
+	@echo '  # how to deploy REANA in development mode, with applicaton'
+	@echo '  # autoreload on code changes and debugging capabilities:'
+	@echo '  $$ CLUSTER_CONFIG=dev make build'
```

Comment:
Until https://github.com/reanahub/reana-workflow-controller/issues/64 is solved, we had better provide the full recipe, going like:

```

$ minikube mount ...
$ cd reana-server && pip install -e . && cd ..  # workaround necessary for r-w-c#64  
$ CLUSTER_CONFIG=make build
$ CLUSTER_CONFIG=make deploy
```

Reply:
You can also add the classical production mode hint before this one:

```
# how to build and deploy REANA in production mode:
$ make build
$ make deploy

# how to build and deploy REANA in developer mode:
$ minikube mount..
$ cd reana-server && ...
$ CLUSTER_CONFIG=make build
$ CLUSTER_CONFIG=make deploy
```


so that the help would tackle the two scenarios.